docs/admin/config-guide/user-password-validation.md

```diff
@@ -0,0 +1,41 @@
+---
+tags:
+  - administration
+  - configuration
+  - password
+---
+
+# Password validation
+User password validation can be set to check the password strength, like the password must have at least one uppercase, or lowercase, number, and the length of the password
+By default the attribute validation is not enabled.  
+
+## Enable validation
+To enable custom validation for password attribute use admin-ui Schema screen.
```

Comment:
`admin-ui` isn't a part of Janssen Server. 

In this PR, We need to talk about how to achieve the same with three config tools that are available.
- CLI
- TUI
- Config-API

Reply:
@ossdhaval have updated the document as per suggestion. Request your review and approval.